packages/polymer-legacy-adapter/src/template-renderer-grid-templatizer.js

```diff
@@ -113,4 +114,4 @@ export class GridTemplatizer extends Templatizer {
   }
 }
 
-customElements.define(GridTemplatizer.is, GridTemplatizer);
+defineCustomElement(GridTemplatizer);
```

Comment:
This class is missing `static get version()` - we might want to add one for consistency.